pkg/codegen/inline.go

```diff
@@ -26,9 +26,11 @@ import (
 
 // GenerateInlinedSpec generates a gzipped, base64 encoded JSON representation of the
 // swagger definition, which we embed inside the generated code.
-func GenerateInlinedSpec(t *template.Template, importMapping importMap, swagger *openapi3.T) (string, error) {
+func GenerateInlinedSpec(t *template.Template, importMapping importMap, swagger *openapi3.T, refNameResolver openapi3.RefNameResolver) (string, error) {
 	// ensure that any external file references are embedded into the embedded spec
-	swagger.InternalizeRefs(context.Background(), nil)
+	if refNameResolver != nil {
+		swagger.InternalizeRefs(context.Background(), refNameResolver)
+	}
```

Comment:
While not perfect we can use the `globalState` to achieve the same thing without breaking the API.
```suggestion
	if globalState.OutputOptions.InternalizeRefs {
		swagger.InternalizeRefs(context.Background(), openapi3.DefaultRefNameResolver)
	}
```